linker/Mono.Linker.Steps/SweepStep.cs

```diff
@@ -130,13 +131,17 @@ void SweepReferences (AssemblyDefinition assembly, AssemblyDefinition target)
 				case AssemblyAction.Copy:
 					// Copy means even if "unlinked" we still want that assembly to be saved back 
 					// to disk (OutputStep) without the (removed) reference
-					Annotations.SetAction (assembly, AssemblyAction.Save);
-					ResolveAllTypeReferences (assembly);
+					Annotations.SetAction (assembly, AssemblyAction.Save);
+					if (!Context.KeepTypeForwarderOnlyAssemblies) {
+						ResolveAllTypeReferences (assembly);
+					}
```

Comment:
I added the above lines in this round of review. There is no reason to resolve all type references if type forwarder-only assemblies are kept.